src/components/Calendar/Calendar.js

```diff
@@ -47,6 +48,8 @@ let components = {
 
 const MyCalendar = props => {
 
+  console.log(props)
```

Comment:
Can you remove the console.log since we shouldn't leave these in when merging to Main?

Reply:
My bad! Just removed it. Good catch!